zk/txpool/pool_zk_limbo.go

```diff
@@ -532,9 +532,9 @@ func (p *TxPool) fromDBLimbo(ctx context.Context, tx kv.Tx, cacheView kvcache.Ca
 			txn.SenderID, txn.Traced = p.senders.getOrCreateID(addr)
 			binary.BigEndian.Uint64(v)
 
-			if reason := p.validateTx(txn, true, cacheView, addr); reason != NotSet && reason != Success {
-				return nil
-			}
+			// if reason := p.validateTx(txn, true, cacheView, addr); reason != NotSet && reason != Success {
```

Comment:
why is this commented out?

Reply:
I think @kstoykov knows why this is commented

Reply:
@kstoykov can comment on this

Reply:
ValidateTx function validates a tx against current network state. Limbo transactions are expected to be invalid according to current network state. That's why there is no point to check it while recovering the pool from a database. These transactions may become valid after some of the current tx in the pool are executed so leave the decision whether a limbo transaction (or any other transaction that has been unwound) to the execution stage.

Reply:
fair enough, but better to add a comment there and remoce the code completely, then it will be okay.